src/Database/LSMTree/Simple.hs

```diff
@@ -539,6 +624,8 @@ Throws the following exceptions:
 ['TableCorruptedError']:
     If the table data is corrupted.
 -}
+-- The worst-case time complexity is \(O(b \log_T n)\).
+-- The amortised time complexity is \(\Theta(b \log\log_T n)\).
```

Comment:
Are these comments about time complexity supposed to stay in the code?

Reply:
Yes.